po/de.po

```diff
@@ -65,50 +64,49 @@ msgid "Install the runner"
 msgstr "Installieren des Ausführungswerkzeugs"
 
 #: src/widgets/info-box.vala:261 src/launchers/steam.vala:621
-#, fuzzy, c-format
+#, c-format
 msgid "The installation of %s was canceled."
-msgstr "Die Installation von "
+msgstr "Die Installation von %s wude abgebrochen."
 
 #: src/widgets/info-box.vala:273 src/launchers/steam.vala:609
-#, fuzzy, c-format
+#, c-format
 msgid "The installation of %s has begun."
-msgstr "Die Installation von "
+msgstr "Die Installation von %s hat begonnen."
 
 #: src/widgets/info-box.vala:287 src/launchers/steam.vala:619
-#, fuzzy, c-format
+#, c-format
 msgid "The installation of %s is complete."
-msgstr "Die Installation von "
+msgstr "Die Installation von %s ist abgeschlossen."
 
 #: src/widgets/info-box.vala:314 src/launchers/steam.vala:577
-#, fuzzy, c-format
+#, c-format
 msgid "The removal of %s is complete."
-msgstr "Die Löschung von "
+msgstr "Die Löschung von %s ist abgeschlossen."
 
 #: src/widgets/info-box.vala:335
-#, fuzzy, c-format
+#, c-format
 msgid "An unexpected error occurred while extracting %s."
-msgstr "Ein unerwarteter Fehler trat auf beim Extrahieren von "
+msgstr "Ein unerwarteter Fehler trat auf beim Extrahieren von %s."
 
 #: src/widgets/info-box.vala:338 src/launchers/steam.vala:623
-#, fuzzy, c-format
+#, c-format
 msgid "An unexpected error occurred while installing %s."
-msgstr "Ein unerwarteter Fehler trat auf beim Installieren von "
+msgstr "Ein unerwarteter Fehler trat auf beim Installieren von %s."
 
 #: src/launchers/bottles.vala:6 src/launchers/steam.vala:6
 msgid "Runners"
 msgstr "Ausführungswerkzeuge"
 
 #: src/launchers/bottles.vala:7 src/launchers/hgl.vala:11
 #: src/launchers/lutris.vala:7
-#, fuzzy
 msgid "Compatibility tools for running Windows software on Linux."
-msgstr "Kompatibilitätstool zum Ausführen von Windows-Spielen unter Linux"
+msgstr "Kompatibilitätstool zum Ausführen von Windows-Spielen unter Linux."
 
 #: src/launchers/bottles.vala:11 src/launchers/hgl.vala:15
 #: src/launchers/lutris.vala:11
-#, fuzzy
 msgid "Vulkan-based implementation of Direct3D 9, 10 and 11 for Linux/Wine."
-msgstr "Vulkan basierte Implementierung von DirectX9, 10 und 11 for Linux/Wine"
+msgstr ""
+"Vulkan basierte Implementierung von DirectX9, 10 und 11 for Linux/Wine."
```

Comment:
Isn't this supposed to be?

`"Vulkan basierte Implementierung von DirectX9, 10 und 11 für Linux/Wine."`

Reply:
"Vulkan basierte Implementierung von Direct3D 9, 10 und 11 for Linux/Wine."

Reply:
Good catch about the Direct3D/DirectX too.